src/blocks/image.js

```diff
@@ -0,0 +1,19 @@
+import { propertiesToText } from './common/base';
+
+const imageToMjml = (item) => {
+    const mjmlProperties = item.properties
+    const properties = propertiesToText(mjmlProperties);
```

Comment:
Any properties to "translate" or "skip"?

Reply:
For example: 'container-background-color'?

Reply:
It would be nice to support border as well (similar to the other components)

Reply:
The only properties that I'm using are `src`, `width`, `alt`, `href` which don't need any translation.

Reply:
Added support for borders.